### PR TITLE
Update readme - Add link to starter template

### PR DESCRIPTION
**Pull request by samruddhikhandale**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,14 +14,10 @@ A **development container** is a running [Docker](https://www.docker.com) contai
 This repository contains a set of **dev container templates** which are source files packaged together that encode configuration for a complete development environment. A template can be used in a new or existing project, and a [supporting tool](https://containers.dev/supporting) will use the configuration from the template to build a development container.
```
Comment (Chuxel) [inline README.md:14]:
```suggestion
This repository contains a set of **Dev Container Templates** which are source files packaged together that encode configuration for a complete development environment. A Template can be used in a new or existing project, and a [supporting tool](https://containers.dev/supporting) will use the configuration from the template to build a development container.
```

File: README.md [hunk 2]
```diff
@@ -14,14 +14,10 @@ A **development container** is a running [Docker](https://www.docker.com) contai
...
 
 ⚠️ Development container 'templates' are a
```
Comment (Chuxel) [inline README.md:16]: At this point, should we just remove this note?  We wouldn't make breaking changes at this point I think.

**Review by Chuxel — APPROVED** [review]
A few minor suggestions otherwise LGTM!

File: README.md [hunk 3]
```diff
@@ -14,14 +14,10 @@ A **development container** is a running [Docker](https://www.docker.com) contai
...
-[**proposed**](https://github.com/devcontainers/spec/blob/main/proposals/devcontainer-templates.md) addition to the
-[development container specification](https://containers.dev/implementors/spec/). **Please note that templates are in
+[**proposed**](https://containers.dev/implementors/templates/) addition to the
+[Dev Container Template specification](https://containers.dev/implementors/spec/). **Please note that templates are in
```
Comment (bamurtaugh) [inline README.md:18]:
```suggestion
[Dev Container Template specification](https://containers.dev/implementors/spec/). **Please note that Templates are in
```

Comment (bamurtaugh) [reply README.md:18]: +1 to what @Chuxel mentions though - perhaps we just remove this section.

File: README.md [hunk 4]
```diff
@@ -32,9 +28,9 @@ specification is accepted, implementation details will be published at
 
 ### Creating your own collection of templates
 
-The [template distribution specification](https://github.com/devcontainers/spec/blob/main/proposals/devcontainer-templates-distribution.md) outlines a pattern for community members and organizations to self-author templates in repositories they control.
+The [Dev Container Template specification](https://containers.dev/implementors/templates-distribution/#distribution) outlines a pattern for community members and organizations to self-author templates in repositories they control.
 
-A [GitHub Action](https://github.com/devcontainers/action) is available to help bootstrap self-authored templates.
+A starter repository [devcontainers/template-starter](https://github.com/devcontainers/template-starter) and [GitHub Action](https://github.com/devcontainers/action) is available to help bootstrap self-authored templates.
```
Comment (bamurtaugh) [inline README.md:33]:
```suggestion
A starter repository [devcontainers/template-starter](https://github.com/devcontainers/template-starter) and [GitHub Action](https://github.com/devcontainers/action) are available to help bootstrap self-authored Templates.
```

**Review by bamurtaugh — APPROVED** [review]
A couple of minor comments, looks great. Thanks!

File: README.md (hunk 2 above)
Comment (samruddhikhandale) [reply README.md]: Makes sense!

File: README.md (hunk 3 above)
Comment (samruddhikhandale) [reply README.md]: Just Removed!

**Review by joshspicer — APPROVED** [review]

**Pull request merged**